src/sections/layout/footer/Footer.tsx

```diff
@@ -9,7 +9,7 @@ export function Footer() {
   const { t } = useTranslation('footer')
 
   return (
-    <footer className={styles.container}>
+    <footer data-testid="footer" className={styles.container}>
```

Comment:
```suggestion
    <footer className={styles.container}>
```
I think we can remove this data-testid now